packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.tsx

```diff
@@ -224,6 +228,7 @@ export class DetailsRow extends BaseComponent<IDetailsRowProps, IDetailsRowState
         style={ { minWidth: viewport ? viewport.width : 0 } }
         aria-selected={ isSelected }
         allowFocusRoot={ true }
+        isInnerZoneKeystroke={ (ev: React.KeyboardEvent<HTMLElement>) => (this.props.innerZoneFocusable ? ev.which === getRTLSafeKeyCode(KeyCodes.down) : false) }
```

Comment:
I still don't think that hardcoding this to `down` is good idea. If really required, let's make it configurable